skimage/util/_invert.py

```diff
@@ -28,6 +38,13 @@ def invert(image):
            [225, 255,   0]], dtype=uint8)
     """
     if image.dtype == 'bool':
-        return ~image
+        return(~image)
```

Comment:
I don't think we use parentheses in the return statements anywhere across the library. Please, avoid using them if possible.